OracleSOASuite/dockerfiles/README.md

```diff
@@ -1,102 +1,37 @@
 SOA on Docker
```

Comment:
This README should be very generic and not go into details of how to run the Admin Server and managed servers.  Each version of SOA 12.2.1.3 and 12.2.1.4 will have its own README. Look for example https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure/README.md

Reply:
This README was created when we released 12.2.1.3 Docker for Dev and Test purpose. This  was written keeping in mind for standalone Docker deployment not with Kubernetes. Since, standalone Docker deployment still being supported, starting Admin and Managed servers instructions are still relevant and in scope. Let me if this is not correct.

Reply:
Now you will have different versions 12.2.1.3 and 12.2.1.4, the README in the OracleSOASuite level should be very generic and just indicate that under dockerfiles we have the Dockerfiles and scripts to build binary images of SOA and OSB and that under samples there are instructions and Dockerfiles to patch these GitHub images.  The details of how to build and run should be in the individual 12.2.1.3 and 12.2.1.4 sub directories.

Reply:
For upper stack we're taking a slightly different patching process than Weblogic. In Weblogic for patch image creation your base image is Weblogic released base image and the you're applying the patch.

Refer to line 23 and 40 
https://github.com/oracle/docker-images/blob/master/OracleWebLogic/samples/12213-patch/Dockerfile

For Upper stack our base image if FMW Infra not the Upper stack released image. We're recreating the image with patch applied. This we did based on imagetool create option and your recommendation that updating is going to add to the size.

Refer to line 17 and 54 

 https://github.com/oracle/docker-images/blob/master/OracleSOASuite/dockerfiles/12.2.1.3/Dockerfile.patch

So, for every release we will have 2 sample Docker files, one without patch and one with patch. Hope this clarifies.

Reply:
Ok I agree with this approach but if this is the case then you need to document this in the README.